ehsm_kms_service/test/test_kms_with_rest.py

```diff
@@ -2,157 +2,146 @@
 import json
 import argparse
 import base64
+import time
+import random
+
+appid = 202112101919
+nonce = ''
+nonce = nonce.join(random.choice('0123456789') for i in range(16))
```

Comment:
It's ranged from 0~0x2386F26FC0FFFF, 6 bytes~7 bytes.

I checked the design doc, where the nonce length is defined as 16. Does the 16 mean 16 Bytes?
If yes, nonce = ''.join(random.choice('0123456789ABCDEF') for i in range(32))
I think 4 bytes of the length is enough: nonce = base64(os.urandom(4))

Reply:
int.from_bytes(os.urandom(4), "big")

Reply:
It will use int.from_bytes(os.urandom(16), "big")